pkg/api/controller.go

```diff
@@ -175,7 +179,9 @@ func (c *Controller) Login(w http.ResponseWriter, r *http.Request, body LoginJSO
 	secret := c.Auth.SecretStore().SharedSecret()
 	// user.Username will be different from username/access_key_id on
 	// LDAP login.  Use the stored value.
-	tokenString, err := GenerateJWT(secret, user.ID, loginTime, expires)
+	// audiance and email set here to be an empty string for backward compatibility
```

Comment:
I don't understand why.  The user has logged in, we have their email.  Why don't we allow them to use it?

Also can cause issues with users having 2 different identities depending on how they choose to login.  This is also relevant to audits conducted by the lakeFS admin, of course.